backend-ssi/app/src/main/kotlin/backend/ssi/blogpost/domain/BlogCategory.kt

```diff
@@ -0,0 +1,5 @@
+package backend.ssi.blogpost.domain
+
+enum class BlogCategory {
+    TECH, SPORT, FASHION, LIFESTYLE, LITERATURE, CARS, TUTORIAL
+}
```

Comment:
U mnie w pracy nie chcą znaczka braku nowej linii i mnie osobiście już trochę razi w oczy, nie wiem czy komuś jeszcze to przeszkadza